ddp-workspace/projects/toolkit/src/lib/components/app/app.component.ts

```diff
@@ -9,15 +9,27 @@ import { WarningComponent } from '../dialogs/warning.component';
 import { CommunicationService } from '../../services/communication.service';
 import { ToolkitConfigurationService } from '../../services/toolkitConfiguration.service';
 import { distinctUntilChanged, filter, map, scan, shareReplay, startWith, switchMap, take, tap } from 'rxjs/operators';
-import { BehaviorSubject, merge, Observable, of, Subscription } from 'rxjs';
-import { BrowserContentService, UserProfile, UserProfileDecorator, UserProfileServiceAgent, WindowRef, RenewSessionNotifier } from 'ddp-sdk';
+import { BehaviorSubject, merge, Observable, of } from 'rxjs';
+import {
+    BrowserContentService,
+    UserProfile,
+    UserProfileDecorator,
+    UserProfileServiceAgent,
+    WindowRef,
+    RenewSessionNotifier,
+    AnalyticsEventsService,
+    CompositeDisposable,
+    AnalyticsEvent
+} from 'ddp-sdk';
 
 declare global {
     interface Window {
         twttr: any;
     }
 }
 
+declare const ga: Function;
```

Comment:
Would be nice if we had @types for Google Analytics .

Reply:
Tried to use types, but didn't find appropriate types in the library https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/types/google.analytics/index.d.ts